Week-2/2.Event-application.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+const day = [
```

Comment:
Nothing else needs day so you can put it inside getEventWeekday

Instead of having getEventWeekday return a number, make it return the day mismo